service/metadata.yaml

```diff
@@ -0,0 +1,68 @@
+type: service
+
+status:
+  class: pkg
+  stability:
+    development: [traces, metrics, logs]
+  distributions: [core, contrib]
+
+tests:
+  goleak:
+    ignore:
+      top:
+        # See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
+        - "go.opencensus.io/stats/view.(*worker).start"
```

Comment:
This is because of ./internal/proctelemetry/config.go only, right?

Reply:
correct